app/mails/management/commands/import.py

```diff
@@ -120,3 +97,45 @@ def handle(self, *args, **kwargs):
                 self.import_mail(message)
 
             imap_conn.expunge()
+
+
+def handle_anti_spam_account(keys, sender, message, account):
+    if account.anti_spam:
+        if not len(keys):
+            message.delete()
+            logger.error("Mail from %s deleted: No key" % sender)
+            tools.send_wrong_recipient_mail(sender)
+
+        return
+    elif not any(key == account.key for key in keys):
+        message.delete()
+        logger.error("Mail from %s deleted: Wrong key" % sender)
+        return
+
+
+def save_received_statistic(delay_addresses, mail, sent_date):
+    for delay_address in delay_addresses:
+        rec_stat = Statistic(
+            type="REC",
+            email=re.sub(r"(^\d+[dmw])(\.[0-9a-z]{10})", r"\1", delay_address),
+        )
+        due = Due(
+            mail=mail,
+            due=sent_date
+            + datetime.timedelta(
+                tools.get_delay_days_from_email_address(delay_address)
+            ),
+        )
+        due.save()
+        rec_stat.save()
+
+
+def save_oblivious_statistic(recipients, mail, delay_addresses):
+    for rec in recipients:
+        recipient = Recipient(mail=mail, email=rec["email"], name=rec["name"])
+        recipient.save()
+        if rec["email"] not in delay_addresses:
+            obl_stat = Statistic(type="OBL", email=rec["email"])
+            obl_stat.save()
+        else:
```

Comment:
The `else: continue` is not really needed, as it's the last step in the loop anyway, so that's what is happening anyway

Reply:
deleted